x-pack/plugins/security_solution/server/endpoint/services/actions/clients/get_response_actions_client.ts

```diff
@@ -33,8 +31,4 @@ export const getResponseActionsClient = (
     case 'sentinel_one':
       return new SentinelOneActionsClient(constructorOptions);
   }
-
-  throw new UnsupportedResponseActionsAgentTypeError(
```

Comment:
Why was this removed?

Reply:
right... its fine to remove it - but what you referred to is a `type` validation. The `throw` here protected against runtime miss-usage (i.e. type casting for `agentType`).

Reply:
I see. I'll undo that change then.

Reply:
a315f7cbebf883ce3ace397c7f09f3821d761932